src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1971,8 +1970,7 @@ internal virtual void AggregateRenderPackages(IEnumerable<HelixRenderPackage> pa
                     //for each instancable item and add instance transforms.  
                     //If we have any mesh geometry that was not associated with an instance, remove the previously added
                     //mesh data from the render package so the remaining mesh can be added to the scene.
-                    if (rp.MeshVertexRangesAssociatedWithInstancing.Any() 
-                        && DynamoModel.FeatureFlags?.CheckFeatureFlag<bool>("graphics-primitive-instancing", false) == true)
+                    if (rp.MeshVertexRangesAssociatedWithInstancing.Any())
```

Comment:
Why aren't you first checking if the `useinstancing` flag is on over here?

Reply:
Ahh yep

Reply:
Hey @aparajit-pratap this one is back.... So I think there is a question to be looked at here.  In general we have fed preference level items that impact how something is tessellated in the TessellationParameters.  Show Edges for example.  We go Preferences -> TessellationParameters -> Tesselate -> Render.  Render doesn't know about TessellationParameters.  If we do the same here then the Render function should not be deciding to use Instancing... It should show use instancing if instancing data is present.  The way the data in the RenderPackage works it actually might render everything wrong if it ignores the data associated with instancing since all the vertices are stored in the same array.